addusers.py

```diff
@@ -212,7 +212,7 @@ def parse_rows(rows, select_user=None):
     """
     # Column index in the Google Sheet for username
     # This may need to be updated if question order on the form is changed
-    USER_COLUMN = 4
+    USER_COLUMN = rows[0].index(user_column)
```

Comment:
This is nitpicky, but it would be better to do the `config.get()` step right here, to minimize how much we are assuming about the environment.  I've been allowing functions in this script to assume they will inherit the `config` variable, but I'd rather not assume too much else.

It would also be good to change `USER_COLUMN` to `user_column` since it's no longer a hard-coded constant, and use some other name ( `user_column_header` or something) for the string value.

Both of the above also apply to PROJECT_COLUMN in the other script.